README.org

```diff
@@ -0,0 +1,143 @@
+** Running Instructions
+To execute the Docker container run 
+#+begin_src shell
+docker-compose build
+docker-compose up
+#+end_src
+
+The command
+#+begin_src shell
+docker-compose up --build
+#+end_src
+may result in:
+#+begin_src shell
+OSError: Can't load tokenizer for 'openai/clip-vit-large-patch14'. If you were trying to load it from 'https://huggingface.co/models', make sure you don't have a local directory with the same name. Otherwise, make sure 'openai/clip-vit-large-patch14' is the correct path to a directory containing all relevant files for a CLIPTokenizer tokenizer.
+#+end_src
+
+** Install CUDA toolkit
+
+Uninstall previous version
+https://docs.nvidia.com/cuda/cuda-installation-guide-linux/#removing-cuda-toolkit-and-driver
+
+Download and install new
+https://developer.nvidia.com/cuda-downloads
+
+** Install nvidia-container-toolkit
+
+#+begin_src shell
+  # Add nvidia toolkit repo to apt get
+  distribution=$(. /etc/os-release;echo $ID$VERSION_ID) \
+      && curl -fsSL https://nvidia.github.io/libnvidia-container/gpgkey | sudo gpg --dearmor -o /usr/share/keyrings/nvidia-container-toolkit-keyring.gpg \
+      && curl -s -L https://nvidia.github.io/libnvidia-container/$distribution/libnvidia-container.list | \
+            sed 's#deb https://#deb [signed-by=/usr/share/keyrings/nvidia-container-toolkit-keyring.gpg] https://#g' | \
+            sudo tee /etc/apt/sources.list.d/nvidia-container-toolkit.list
+  # update and install 
+  sudo apt-get update
+
+  # install nvidia-cotainer-toolkit
+  sudo apt-get install -y nvidia-container-toolkit
+
+  # check version 
+  nvidia-ctk --version
+
+  # Generate a CDI specification that refers to all devices
+  sudo nvidia-ctk cdi generate --output=/etc/cdi/nvidia.yaml
+
+  # Check name of the created devices
+  sudo grep "  name:" /etc/cdi/nvidia.yaml
+#+end_src
+
+Install Docker
+#+begin_src shell
+curl https://get.docker.com | sh \
+  && sudo systemctl --now enable docker
+#+end_src
+
+Configure the Docker daemon to recognize the NVIDIA Container Runtime:
+#+begin_src shell
+sudo nvidia-ctk runtime configure --runtime=docker
+
+# Restart Docker
+sudo systemctl restart docker
+#+end_src
+
+If after all this process the following error appears:
+#+begin_src shell
+docker: Error response from daemon: could not select device driver "" with capabilities: [[gpu]].
+#+end_src
+
+Repeat this steps:
+#+begin_src shell
+sudo nvidia-ctk runtime configure --runtime=docker
+
+# Restart Docker
+sudo systemctl restart docker
+#+end_src
+
+*** Refs
+[[Nvidia Container Toolkit Installation Guide][https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html#installation-guide]]
+
```

Comment:
```suggestion
- [*ADD DOCKER INSTALL STEPS REF*]

```

Reply:
https://docs.docker.com/engine/install/ubuntu/#install-using-the-convenience-script